src/main/java/net/mcreator/minecraft/api/ModAPI.java

```diff
@@ -38,21 +38,7 @@ void setImplementations(Map<String, Implementation> implementations) {
 		this.implementations = implementations;

```

Comment:
ModAPI itself could also be a record

Reply:
:heavy_check_mark: Fixed
But I also moved Implementation to a separate class file to make it look nice.